features/admin/quota.feature

```diff
@@ -177,6 +182,7 @@ Feature: Quota related scenarios
   # @case_id OCP-11566
   @admin
   @inactive
+  @ppc64le
```

Comment:
for inactive cases, we never verified they work on other architectures than amd64, and we usually skip these in the CI. Are you sure to add them?

Reply:
if it's inactive it's not going to be run...so there's probably no need to add it.

Reply:
I've added the ppc64le tag only where there are other architecture labels present